packages/cbioportal-utils/src/signal/SignalMutationUtils.ts

```diff
@@ -195,3 +196,16 @@ export function calculateTotalVariantRatio(
 ) {
     return totalVariants(counts1) / totalVariants(counts2);
 }
+
+export function getMutationStatus(mutation: SignalMutation) {
```

Comment:
maybe rename this as `getSignalMutationStatus`? Theoretically we may have a `getMutationStatus` function in `MutationUtils` too. There would be an export conflict in that case. Not a big deal though, we can always rename functions when we have such conflicts.

```suggestion
export function getSignalMutationStatus(mutation: SignalMutation): SignalMutationStatus {
```